resolwe_bio/descriptors/unified-descriptor.yml

```diff
@@ -0,0 +1,1877 @@
+          type: basic:string
+          required: false
+          choices:
+            - label: Pretreatment
+              value: pretreatment
+            - label: Posttreatment
+              value: posttreatment
+        - name: tumor
+          label: Tumor
+          type: basic:string
+          description: |
+            Select what molecule your samples are from.
+          required: false
+          choices:
+            - label: RNA
+              value: rna
+            - label: DNA
+              value: dna
+        - name: normal
+          label: Normal
+          type: basic:string
+          description: |
+            Select what molecule your samples are from.
+          required: false
+          choices:
+            - label: RNA
+              value: rna
+            - label: DNA
+              value: dna
+        - name: date_of_preparation
+          label: Date of preparation
+          type: basic:date
+          required: false
+        - name: date_of_shipping
+          label: Date of shipping
+          type: basic:date
+          required: false
+        - name: date_received
+          label: Date received
+          type: basic:date
+          required: false
+        - name: platform
+          label: Platform
+          type: basic:string
+          description: |
+            Select by which platform the sample was prepared.
+          required: false
+          choices:
+            - label: RNA-seq
+              value: rna-seq
+            - label: DNA-seq
+              value: dna-seq
+            - label: Nanostring
+              value: nanostring
+            - label: Microarray
+              value: microarray
+            - label: Edge-seq
+              value: edge-seq
+        - name: cro
+          label: CRO
+          type: basic:string
+          description: |
+            Name of the Contract Research Organisation that worked on the samples.
+          required: false
+        - name: description_of_sample_preparation
+          label: Description of sample preparation
+          type: basic:string
+          required: false
+    - name: cell_line_information
+      label: Cell line information
+      group:
+        - name: cell_line_bank
+          label: Cell line Bank
+          type: basic:string
+          description: |
+            Cell line Banks, e.g. ATCC, ECACC, etc.
+          required: false
+        - name: cell_line_bank_id
+          label: Cell line Bank ID
+          type: basic:string
+          description: |
+            Official cell line ID.
+          required: false
+        - name: characteristics
+          label: Characteristics
+          type: basic:string
+          required: false
+        - name: growth_protocol
+          label: Growth Protocol
+          type: basic:string
+          description: |
+            Describe the conditions that were used to grow or maintain cells prior to extract preparation.
+          required: false
```

Comment:
Should this be seperated from `growth_protocol` in the group `general`?

Reply:
@OtonicarJan This should be moved to the group of `cell_line_information` as in https://github.com/genialis/genialis-bio/blob/master/genialis_bio/inhibrx/descriptors/Inhibrx-cell-line.yml